src/main/java/org/mockito/internal/junit/JUnitRule.java

```diff
@@ -14,25 +13,74 @@
 public class JUnitRule implements MockitoRule {
 	
 	private final MockitoLogger logger;
+    private final boolean silent;
 
-	public JUnitRule(MockitoLogger logger) {
+    public JUnitRule(MockitoLogger logger, boolean silent) {
```

Comment:
+1 but I want to wait for some feedback. @ChristianSchwarz already had a good idea for "strict" mode that fails when unusued stubbing is detected.